### PR TITLE
EKF: improve reporting of filter status

### PR DESCRIPTION
**Pull request by priseborough**

### CODE REVIEW

File: EKF/ekf_helper.cpp [hunk 1]
```diff
@@ -316,6 +316,12 @@ void Ekf::get_heading_innov_var(float *heading_innov_var)
 	memcpy(heading_innov_var, &_heading_innov_var, sizeof(float));
 }
 
+// get GPS check status
+void Ekf::get_gps_check_status(uint16_t *val)
+{
+	memcpy(val, &_gps_check_fail_status, sizeof(uint16_t));
```
Comment (LorenzMeier) [inline EKF/ekf_helper.cpp]: the better solution here is: ```*val = _gps_check_fail_status```

Comment (priseborough) [inline EKF/ekf_helper.cpp]: Done

**Comment by priseborough** [comment]
Any reason why this should not be pulled in?

**Comment by LorenzMeier** [comment]
No, let's do it!

**Pull request merged**